luaui/Widgets/gui_info.lua

```diff
@@ -1766,6 +1782,33 @@ function widget:DrawScreen()
 	end
 end
 
+-- If the time to finish a full burst is greater than the reload time,
+-- then the units reload time is irrelevant and will continuously fire.
+function hasContinuousFire(burstCount, burstRate, reloadTime)
+	local burstRate = round(burstRate, 3)
+	local totalBurstTime = burstCount * burstRate
+
+	return totalBurstTime >= tonumber(round(reloadTime, 3))
```

Comment:
why the rounding?

Reply:
I am using rounding to 3 decimals as a tolerance for the comparison since these are floats. Although I should only be rounding in the comparison not before the totalBurstTime calculation. Ill fix that.